content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -213,11 +221,41 @@ spec:
       ports:
         http: 8080
       labels:
-        app: vm1
+        app: vmhttp
         version: 1
```

Comment:
Did you actually run this command and verify that it works? When I tried it, I got this error output: "json: cannot unmarshal number into Go value of type string". Removing "version: 1" fixed the problem.

Reply:
I should add, while removing "version: 1" fixed the error I was seeing, I don't actually know if that was the correct thing to do. No idea why "version: 1" is there to begin with...

Reply:
A version label is optional. It would be used if the endpoint represents a specific version of the service that we want to use in subsets for routing.

The proper fix is to put quotes around the number value:
```
version: "1"
```

Reply:
I tried this version 1 stuff and not working due to this string/numeric stuff as you mentioned.

@frankbu  I agreed with @justinburke that we should make a minimal working tutorial setup. And we don't have any follow-up steps that depend on this version/routing, stuff.

I think we can remove it.

Reply:
@incfly if you want the simplest possible tutorial, then you can remove the app label too.

Either way, the text just above says: `Each ServiceEntry configuration contains the IP addresses, ports, and labels (where appropriate) ...`, so I think the `labels (where appropriate)` part needs some clarification about what that means. It means needed for route rules to use, which usually includes some kind of version label.

Reply:
Got it, thanks for explaination. I changed "1" to "v1" and verified this cmd now works.